mindjunkies/payments/models.py

```
@@ -44,6 +44,36 @@ class Meta:
         return self.user.username + " - " + self.course.title
+    
+
+
+
+class Balance(BaseModel):
+    user = models.OneToOneField(
+        User, on_delete=models.CASCADE, related_name="balance"
+    )
+    amount = models.DecimalField(max_digits=10, decimal_places=2, default=0)
+    last_updated = models.DateTimeField(auto_now=True)
+    
```

Ruff (0.8.2)

Blank line contains whitespace

Remove whitespace from blank line

(W293)